### DISCUSSION TITLE
I know 11th gen Intel are not supported currently, but ...

### DISCUSSION THREAD

**Discussion by SoFLazerTag** [sebanc/brunch#774] [General]
Can you elaborate on what is required to support them or possibly when support could get added?

Is it about waiting until the recovery images on such units is made available? Plus Brunch changes?

If you seek anyone for testing I would love to when the time comes. I'm now rocking the Lenovo Series 7 Yoga with a i5-1135G7 and really enjoying it.  Would enjoy it way more with Chrome OS ;)  For the record I am loving Brunch to get Chrome OS added on a couple different laptops for my friends and it works great.  Thank you!

**Comment by Warrentheo** [comment]
I don't know for sure, but this is most likely related to the Intel Iris XE integrated graphics...  Intel put out a much better graphics card that you can almost do mid to high end gaming on, but the old Intel drivers in the Linux Kernel will probably need a few updates to get that working in Android...

**Comment by Kernel511** [comment]
Intel Iris XE is supported from Kernel 5.11 onwards

**Comment by Warrentheo** [comment]
I thought something similar about Nvidia graphics cards, this is what Sebanc had to say on that...

http://github.com/sebanc/brunch/discussions/871

**Comment by SoFLazerTag** [comment]
I see 'volteer' is now listed on the Google recovery images page, but no update yet to download.  This will be pretty important for anyone with Iris GPU as I assume it includes that driver now. 
 Actually, does anyone know if any other boards have the Intel Iris driver already?  I know that rammus and hatch currently do not.  Haven't bothered with others so far.  Anyway, just curious.

**Comment by SoFLazerTag** [comment]
volteer image dropped overnight on cros serving.  Happy to report I do finally get past the Chrome logo but it does seem to simply get stuck on white screen showing the 'Shut Down' button to the lower left and then the Chrome dock lower right with wifi/battery icon and time.  Time doesn't change and nothing can be clicked on at that point.  I did notice if I mouse around the center of the screen the pointer changes to a pointing finger, but you can't see anything there.  Also from console I do see the iris_dri.so driver now as well, I assume why I'm not stuck at a logo any longer.

If any new updates start coming out for brunch and you would like some testers for it I am totally ready to help.

thanks

EDIT: I am a dummy, totally forgot to try switching to the 5.10 kernel. No hang on it and all logged in and checking things out now!
